src/js/views/patients/sidebar/action/action-sidebar-action_views.js

```diff
@@ -130,12 +130,16 @@ const ActionView = View.extend({
     duration: '[data-duration-region]',
   },
   modelEvents: {
+    'change:details': 'onChangeDetails',
     'change:_state': 'onChangeActionState',
     'change:_owner': 'onChangeOwner',
     'change:due_date': 'onChangeDue',
     'change:due_time': 'onChangeDue',
     'change:duration': 'onChangeDuration',
   },
+  onChangeDetails() {
+    this.showEditForm();
+  },
```

Comment:
A user could be making edits to the action details in the action sidebar and have those edits wiped out and replaced if a `DetailsChanged` web socket message comes in.

Is that okay from a UI/user experience perspective?

Reply:
So this might be tough.. but if the user is currently modifying the details we do not need to rerender the form.. we can just let them override them, but if they cancel the form, it should show the socket details.

Reply:
Okay ya that seems like the way to go. See if I can get that working.

Reply:
Just pushed a commit with a potential implementation.